src/Server.cpp

```diff
@@ -53,6 +53,14 @@ HttpResponse    Server::handleRequest(const HttpRequest& req)
     // Check if a valid route has been found
     if (!route) return response_error( req, res, _cfg, route, 404);
 
+    if ( route->is_redirect ) {
+        req.set_target( route->redirect_to );
+        res = Server::handleRequest( req );
```

Comment:
Why do we set the target and run handleRequest again? Why do we not just have the last 2 lines? The target redirect might not be for this server.